src/ape/api/accounts.py

```diff
@@ -73,19 +73,27 @@ def sign_transaction(self, txn: TransactionAPI) -> Optional[TransactionSignature
           :class:`~ape.types.signatures.TransactionSignature` (optional): The signed transaction.
         """
 
-    def call(self, txn: TransactionAPI, send_everything: bool = False) -> ReceiptAPI:
+    def call(
+        self,
+        txn: TransactionAPI,
+        send_everything: bool = False,
+        raise_on_fail: bool = True,
```

Comment:
By changing the API method, it will technically be a breaking change.
I was going to see if we could hold off on changing the actual API until we are ready for 0.5... Are API changes required to get the behavior we need? Is there a short-term solution we could implement?

Reply:
@unparalleled-js The method itself isn't the breaking change is it? Since it provides a default existing calls to it shouldn't be a problem - but the breaking change is the new behavior.

When I spoke with @fubuloubu the way it was described to me was that it should be raising exceptions by default, which is far different from current behavior (can currently raise on gas estimation issues, but doesn't raise if the transaction goes through and ends up reverting)

The way that this wouldn't be breaking is if the default was `raise_on_fail=False`, but only in its current state. When `raise_on_fail` is implemented to account for gas estimation then the default being `False` will change the current behavior as well 👀

Reply:
Oh, cool! I think you're right. I think it only makes `mypy` mad which is nbd.

Reply:
@unparalleled-js ohh true

One issue I did run into for a sec the other day was at first I had args of subclasses in a different order, the type checker was very mad 😂 should be all good now!